user_manual/files/quota.rst

```diff
@@ -3,7 +3,7 @@ Storage quota
 =============
 
 Your Nextcloud admin has the option to set a storage quota on users. Look at 
-the top of your Personal page to see what your quota is, and how much you have 
+the of your Personal page to see what your quota is, and how much you have 
```

Comment:
@Valdnet see here, the sentence is due to your change:

```
Look at the of your Personal page to see what your quota is, and how much you have used.
```

Which is not a proper sentence.

Did you forget to push your second change maybe?